frontend/www/js/omegaup/components/common/Navbar.vue

```diff
@@ -444,6 +448,7 @@ export default class Navbar extends Vue {
   @Prop() isUnder13User!: boolean;
   @Prop() mentorCanChooseCoder!: boolean;
   @Prop() userVerificationDeadline!: Date | null;
+  @Prop() logoutModalVisible: boolean = false;
```

Comment:
In Vue, component properties are immutable, which means that most of the time they should not be able to modify their value.

To handle this, we can use dynamic data. You can declare it as follows, and it should work:
```suggestion
  logoutModalVisible = false;
```

Right below line 454.